chargebee-js-react/src/components/CardCVV.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import Element, { ElementProps } from './Element';
+import { ComponentContext } from "./ComponentGroup";
+
+// @to-update
+export interface CardCVVProps {
+    onBlur: React.MouseEventHandler;
```

Comment:
Please extend this type to include the prop types from component `Element`

Reply:
Done.